src/main/resources/options/options-id-value-transform.json

```diff
@@ -89,5 +88,14 @@
     "potentialAllegationChecklist_13": "There is likely to be a need for the involvement of Intervenors",
     "potentialAllegationChecklist_14": "The case involves an insolvency issue",
     "notApplicable": "Not applicable"
+  },
+  "consentNatureOfApplicationChecklist": {
+    "periodicalPaymentOrder": "Periodical Payment Order",
+    "Lump Sum Order": "Lump Sum Order",
```

Comment:
possibly just a pattern we already use but transforming to same value doesn't do anything

Reply:
Fair point, I was fleshing it out for all values. I can remove the redundant ones so its clearer that these are the 2 fields being mapped :)